__tests__/app.test.js

```diff
@@ -73,6 +73,82 @@ describe("App", () => {
           });
         });
     });
+    // Query tests
+    test("200: Responds with the array of review objects, all of which only have the category of 'dexterity'", () => {
```

Comment:
Please keep your test descriptions a bit more generic. We're not testing specifically for 'dexterity', we're testing to make sure that only reviews with the queried category come back (you've used dexterity to exemplify this in your test, but it won't _always_ be dexterity.

Reply:
Please also test that the category defaults to something if no category is passed.